flutter/integration_test/expected_accuracy.dart

```diff
@@ -23,17 +23,17 @@ const Map<String, Interval> _objectDetection = {
 const Map<String, Interval> _imageSegmentation = {
   'CPU': Interval(min: 0.8387, max: 0.8388),
   'NNAPI': Interval(min: 0.4836, max: 0.4872),
-  'CoreML': Interval(min: 0.9887, max: 0.9888),
-  // accuracy in emulator is 0.83, on real device 0.98
-  'ANE': Interval(min: 0.8387, max: 0.9891),
+  'CoreML': Interval(min: 0.80, max: 0.8275),
+  'ANE': Interval(min: 0.8277, max: 0.8388),
```

Comment:
The current key is not clear enough which backend is used. Maybe we should add vendor name to the key, so it has the format `<vendor>_<accelerator>`?

Reply:
Right now we assume that if different backends use the same accelerator, they should have the same accuracy.
We don't have too big dataset yet but we have an example of pixel backend, which has only 1 exception.
https://github.com/mlcommons/mobile_app_open/blob/c52d0e44ae08c6c05390c16f409df2e14d0a7fd7/flutter/integration_test/expected_accuracy.dart#L18
If we decide to specify backend for each accuracy interval we will increase code size without achieving much.

Reply:
Maybe we can simply add documentation in the beginning of the file, describing accelerators but I'm not sure if it will be helpful.

Reply:
I created another issue to rename the fields (https://github.com/mlcommons/mobile_app_open/issues/540). Maybe it will be more clear after that.